src/main/java/io/cdap/plugin/Trash.java

```diff
@@ -53,6 +54,11 @@ public Trash(ThrashConfig config) {
   @Override
   public void prepareRun(BatchSinkContext context) throws Exception {
     context.addOutput(Output.of(config.referenceName, new ThrashOutputFormatProvider()));
+
+    Schema schema = context.getInputSchema();
+    if (schema != null && schema.getFields() != null) {
+      recordLineage(context, config.referenceName, schema, "Write", "Wrote to Trash.");
```

Comment:
Let's put this method in `ReferenceBatchSink` instead of cdap

Reply:
Done. https://github.com/cdapio/hydrator-plugins/pull/1071